rust/yarp-sys/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "yarp-sys"
+version = "0.1.0"
+edition = "2021"
+license-file = "../../LICENSE.md"
+repository = "https://github.com/ruby/yarp"
+description = "Rust bindings to Ruby's YARP parsing library"
```

we should add a `links` directive:

```suggestion
description = "Rust bindings to Ruby's YARP parsing library"
links = "yarp"
```

---

You know, there was a reason why I specifically didn't do that, but I can't remember what that was... Probably some misinformation I came across in a post somewhere. In any case, I have `build.rs` linking against `rubyparser` (`yarp/build/librubyparser.a`, to be specific), since that was the only static lib I came across--should I be pointing to something else?? FWIW, on macOS, I get a `yarp/lib/yarp.bundle`, but no `libyarp.dylib` anywhere...

---

Weird... my understanding of `links` must be off. If I add `links = "rubyparser"`, I get:

```
error: could not find native static library `rubyparser`, perhaps an -L flag is missing?
```

...but adding `links = "yarp", results in no build errors. ...but even weirder, I also get no build errors if I add `links = "yarpasdfasdf"`. 🤔